selfdrive/car/torque_data/override.toml

```diff
@@ -43,6 +43,7 @@ legend = ["LAT_ACCEL_FACTOR", "MAX_LAT_ACCEL_MEASURED", "FRICTION"]
 "CADILLAC_ESCALADE_ESV_2019" = [1.15, 1.3, 0.2]
 "CADILLAC_XT4" = [1.45, 1.6, 0.2]
 "CHEVROLET_BOLT_EUV" = [2.0, 2.0, 0.05]
+"CHEVROLET_MALIBU_CC" = [1.85, 1.85, 0.075]
```

Comment:
Can you please document where these numbers came from?

Reply:
Self learned values for test drives